gateway/src/proxy.rs

```diff
@@ -121,8 +121,8 @@ async fn proxy(
     let mut res = PROXY_CLIENT
         .call(addr.ip(), &target_url, req)
         .await
-        .map_err(|e| {
-            error!(error = ?e, "gateway proxy client error");
+        .map_err(|err| {
+            error!(error = ?err, "gateway proxy client error");
```

Comment:
Not `dyn StdError` this time?

Reply:
This is one I tried to change, but it was tricky, due to `ProxyError` from the reverse proxy crate not implementing std Error. There are a few more similar cases we still need to tweak to get them covered by error tracking.